src/Features/Blockcore.Features.WebHost/WebHostSettings.cs

```diff
@@ -28,6 +28,21 @@ public class ApiSettings
         /// <summary>URI to node's API interface.</summary>
         public Timer KeepaliveTimer { get; private set; }
 
+        /// <summary>
+        /// If true then the node will add and start the SignalR feature. This should never be enabled if node is accessible to the public.
```

Comment:
Are they on by default? should we document that?

Reply:
this.EnableWS = config.GetOrDefault<bool>("enableWS", false, this.logger);
this.EnableUI = config.GetOrDefault<bool>("enableUI", true, this.logger);
this.EnableAPI = config.GetOrDefault<bool>("enableAPI", true, this.logger);

Probably document somewhere, and we should decide what we want on by default.